docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -0,0 +1,104 @@
+apartment.
+
+This is a simple example, usually {regression} problems are multi-dimensional, 
+so the relationships that {reganalysis} tries to find are between multiple 
+{feature-vars} and the {depvar}. To extend our example, a more complex 
+{reganalysis} could take into account further {feature-vars} like the location 
+of the apartment in the city, on which floor it is, and whether the apartment 
+has a riverside view or not and so on.
+
+
+[discrete]
+[[dfa-regression-features]]
+=== {feature-vars-cap}
+
+{feature-vars-cap} are the values that the {depvar} value depends on. If one or 
+more of the {feature-vars} changes, the {depvar} value also changes. There are 
+three different types of {feature-vars} that you can ship to the {regression} 
+algorithm:
+* Numerical. In our example, the size of the apartment was a 
+  numerical {feature-var}.
+* Categorical. A variable that can have one value from a set of values. The 
+  value set has a fixed and limited number of possible items. In the example, 
+  the location of the apartment in the city (borough) is a categorical variable.
+* Boolean. The riverside view in the example is a boolean value because an 
+  apartment either has a riverside view or doesn't have one.
+Arrays are not supported.
+
+
+[discrete]
+[[dfa-regression-supervised]]
+=== Training the {regression} algorithm
+
+{regression-cap} is a supervised machine learning process, which means that you 
+need to supply a labeled training dataset that has some {feature-vars} and a 
+{depvar}. The {regression} algorithm learns the relationships between the 
+features and the {depvar}. Once you've trained the model on your training 
+dataset, you can reuse the knowledge that the model has learned about the 
+relationships between the data points to make certain inferences about new data 
+points.
+
+The relationships between the {feature-vars} and the {depvar} is described as a 
+mathematical formula. {reganalysis-cap} tries to find the best value for the 
+{depvar} by combining various {ml} techniques, that means that in the {stack}, 
+we use an ensemble learning model that uses multiple algorithms. The performance 
+of an ensemble is usually better than the performance of an individual 
+algorithm because each of the algorithms has a different bias, so each of them 
+falls into different traps. When they are used together as a group, then they 
+capture the characteristics of the data better. The ensemble learning technique 
+that we use in the {stack} is a type of boosting called extreme gradient boost 
+(XGboost) which combines decision trees with gradient boosting methodologies.
+
+ 
+[discrete]
+[[dfa-regression-evaluation]]
+=== Measuring model performance
+
+You can measure how well the model has performed on your training dataset by 
+using the `regression` evaluation type of the evaluate {dfanalytics} API. The 
+mean squared error value that the evaluation may provide you on the training 
```

Comment:
Since this is the first time we're using this term, the acronym should be defined here:
```suggestion
mean squared error (MSE) value that the evaluation may provide you on the training 
```